### DISCUSSION TITLE
WorkflowsToInject

### DISCUSSION THREAD

**Discussion by pappasa** [microsoft/RulesEngine#527] [Q&A]
I see that the class RulesEngine.Models.Rule has the property WorkflowsToInject. Can you describe, what this property does, and if possible give an example?

**Comment by suntang27** [comment]
WorkflowsToInject combine rules from another workflow into the current one. i.e. 

inputWorkflow.json
{
    "WorkflowName": "inputWorkflow",
	"WorkflowsToInject" : ["testWorkflow"],
    "Rules": [

      {
        "RuleName": "DiscountResult",
        "Operator": "OrElse",
        "Rules": [..]
}
]
		
testWorkflow.json
{
    "WorkflowName": "testWorkflow",
    "Rules": [

      {
        "RuleName": "testDiscountResult",
        "Operator": "OrElse",
        "Rules": [..]
}
]
}
          

var workflowRules = new string[] { inputWorkflow.json content, testWorkflow.json content };
{
    CustomActions = new Dictionary<string, Func<ActionBase>>{
                                          {"MyCustomAction", () => new MyCustomAction() }
                                      }
};
var re = new RulesEngine.RulesEngine(workflowRules, reSettings);
